OOModel/src/expressions/MetaCallExpression.cpp

```diff
@@ -62,4 +69,61 @@ MetaDefinition* MetaCallExpression::metaDefinition()
 	return ret;
 }
 
+void MetaCallExpression::bindMetaCalls(Model::Node* n, MetaBinding* binding)
+{
+	if (auto metaCall = DCast<MetaCallExpression>(n))
+		if (auto callee = DCast<ReferenceExpression>(metaCall->callee()))
+			for (auto i = 0; i < binding->mappings()->size(); i++)
+				if (callee->name() == binding->mappings()->at(i)->name())
+					callee->setName(binding->mappings()->at(i)->value()->name());
```

Comment:
To be honest, we shouldn't sprinkle the code with `begin/endModification`. There should be a single, well defined method that starts the whole process. That method should contain the only `begin/end` pair. This also means that the method can't be recursive. Do we have such an entry point to the generation? If not, we should implement one and perhaps, hide some of the other methods (make private/friends) to avoid accidental calling.